gsmphys/sfcsub.F

```diff
@@ -534,6 +546,7 @@ subroutine sfccycle(lugb,len,lsoil,sig1t,deltsfc
 !       rec. 16    canopy water content (cnpanl)  -----> f10m
 !       rec. 17    vegetation type
 !       rec. 18    soil type
+!       rec. 18    soil color ? add later?
```

Comment:
Sure happy to update this—why 22 and not 19?  I didn't fully grok what the original comment was getting at so I just copied this change directly from https://github.com/ufs-community/ccpp-physics/pull/78https://github.com/ufs-community/ccpp-physics/pull/78.

Reply:
I changed it to 19 in 23aeaee and incremented the subsequent numbers, but that could be misguided.  Let me know if you understand the intent of this comment better than me.

Reply:
My bad. I thought the recode number ended at 21, so that's why I said 22. I believe these numbers have something to do with a data format that is now obsolete.

Reply:
Ah gotcha, thanks.  I'll switch back and use 33.

Reply:
Oops just missed @lharris4's merge!  Oh well, I don't think this is super important given that the file format is no longer used.